compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -110,6 +110,19 @@ static LogicalResult verifyLoweringConfiguration(
   return failure(walkResult.wasInterrupted());
 }
 
+static bool hasPackMatmulLhsOp(ModuleOp moduleOp) {
```

Comment:
could we limit this to the dispatch function only?

Reply:
This is only used by KernelConfig.cpp now.